### ISSUE TITLE
Implement setting the reminder

### ISSUE THREAD

**Issue by raimund-schluessler** [owncloud/tasks#314]
Also send out a notification when a reminder is due.

**Comment by raimund-schluessler** [comment]
As stated in https://github.com/owncloud/tasks/issues/251:

The new notification API owncloud/core#18699 and app https://github.com/owncloud/notifications coming with oC8.2 (?) should be used to remind the user on a due reminder.

TODO:

    create a Cron job which checks the database for due reminders ?
    hand over notifications to the notification API
    create the prepare() method

**Comment by 4oo4** [comment]
Any update on where this is at? I recently upgraded from owncloud 8.2 to Nextcloud 10 (upgraded to NC 9.0 in between), and it seems that the reminder function was removed. I was poking through the source and it looks like the code for reminders is still there in `templates/part.details.php` and `js/public/app.js` but is commented out.

I tried to re-enable it by uncommenting that code, but that didn't seem to work. Is there a reason that it's commented out, and is there a plan to bring that back? I used that feature a lot and miss it now that I've upgraded.

Thanks
